rest_tools/client/client.py

```diff
@@ -184,6 +210,39 @@ def request_seq(self, method, path, args=None):
         finally:
             self.session = s
 
+    def request_stream(
+        self,
+        method: str,
+        path: str,
+        args: Optional[Dict[str, Any]] = None,
+        chunk_size: Optional[int] = 8096
+    ) -> Generator[JSONType, None, None]:
+        """Send request to REST Server, and stream results back.
+
+        `chunk_size=None` will read data as it arrives
+        in whatever size the chunks are received.
+
+        Args:
+            method (str): the http method
+            path (str): the url path on the server
+            args (dict): any arguments to pass
+            chunk_size (int): chunk size (see above)
+
+        Returns:
+            dict: json dict or raw string
+        """
+        s = self.session
+        try:
+            self.open(sync=True)
+            url, kwargs = self._prepare(method, path, args)
+            resp = self.session.request(method, url, stream=True, **kwargs)
+            resp.raise_for_status()
+            for line in resp.iter_lines(chunk_size=chunk_size):
+                yield self._decode(line)
```

Comment:
Should the line be stripped of whitespace?  Not sure if that matters, though it could raise an error on an otherwise empty line.

Reply:
I think `iter_lines` takes care of that. But I'll test it.